Pages/ManageIndividualVolunteer.razor

```diff
@@ -0,0 +1,106 @@
+﻿@page "/volunteer/{Id:guid}"
+@using System.Linq
+@using Karma.Models
+@inject IJSRuntime m_jsRuntime
+@using Microsoft.EntityFrameworkCore;
+
+@if (m_httpContextAccessor.HttpContext.User.Identity.IsAuthenticated)
+{
+    @if (volunteer != null)
+    {
+        <h1>Manage volunteer</h1>
+
+        <label for="fname">Volunteer name:</label>
+        <input type="text" @bind="@volunteer.Name" @oninput="@((e) => { volunteer.Name = (string)e.Value; })" />
+        <br>
+        <br>
+        <label for="fname">Volunterr surname:</label>
+        <input type="text" @bind="@volunteer.Surname" @oninput="@((e) => { volunteer.Surname = (string)e.Value; })" />
+        <br>
```

Comment:
Afaik having many newlines like this is not considered good. Let's create an issue to fix this in the future.